google/cloud/sql/connector/connector.py

```diff
@@ -48,7 +49,12 @@ def _get_keys(loop: asyncio.AbstractEventLoop) -> concurrent.futures.Future:
     return _keys
 
 
-def connect(instance_connection_string: str, driver: str, **kwargs: Any) -> Any:
+def connect(
+    instance_connection_string: str,
+    driver: str,
+    ip_types: List[IPTypes] = DEFAULT_IP_TYPES,
```

Comment:
I'll defer to your judgement on this. I know that we originally had the list so that users could set a fallback option ("if private is unavailable, connect to public" but I'm not sure if there's a real-world use case for that

Reply:
IMO, it's confusing because we only "fall back" when the IP doesn't exist - not when it's unavailable. If an instance has a private IP and a public IP, and they specify "PRIVATE,PUBLIC" it'll always use the Private IP. But I'm open to your thoughts on it.

Reply:
in that case, I agree with you, let's simplify it. I doubt anyone using this library wouldn't know whether the IP address they want to use exists, and if it doesn't exist and they expect it to, they should get error feedback right away